hyperbahn-staging.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+set -e
+
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
```

Comment:
`$(dirname $(readlink -f $0))` but only on GNU system... maybe what you have is The Way 
™ for Portability ™.